### DISCUSSION TITLE
Build Penpot Library for AstroUXDS

### DISCUSSION THREAD

**Discussion by memtech3** [RocketCommunicationsInc/astro#1295] [Feature Requests]
Howdy,

I love the work y'all have done with AstroUXDS and want to use it in an open source project. Have y'all considered supporting penpot, a free and open source design tool like Figma? 

Maintaining a penpot library in addition to the Figma library would make it much easier for small community and hobby projects (like mine) to build on top of Astro's work.

**Comment by dmcalester** [comment]
Hi there,

Yes, PenPot has been on our radar for a while. It hasn’t made it on to the product road map just yet, but it’s quite high on the list of things we’d like to add for a future Astro release. Demand signal from the user community is one of our biggest factors in getting a feature on the road map, particularly if that project aligns with other community needs. If you are able to discuss your project please drop us a line support@astrouxds.com.
